code/modules/mining/mine_items.dm

```diff
@@ -58,7 +58,12 @@
 	var/digspeed //moving the delay to an item var so R&D can make improved picks. --NEO
 	origin_tech = list(TECH_MATERIAL = 1, TECH_ENGINEERING = 1)
 	attack_verb = list("hit", "pierced", "sliced", "attacked")
-	var/drill_sound = 'sound/weapons/chisel1.ogg'
+	var/drill_sound = list ('sound/items/pickaxe/mine1.ogg',
```

Comment:
This will crash the proc if the playing of the sound is not altered. You cannot trivially replace 1 sound instance with a list of sounds.